src/libs/import.c

```diff
@@ -450,6 +455,8 @@ static GtkWidget *_lib_import_get_extra_widget(dt_lib_module_t *self,dt_lib_impo
     gtk_toggle_button_set_active(GTK_TOGGLE_BUTTON(ignore_jpeg),
                                  dt_conf_get_bool("ui_last/import_ignore_jpegs"));
     gtk_box_pack_start(GTK_BOX(extra), ignore_jpeg, FALSE, FALSE, 0);
+    g_signal_connect(G_OBJECT(ignore_jpeg), "clicked",
```

Comment:
The fix does look sane, and is basically the same as i expected.
I'm sure that you have troughly tested this, and it fixes the bug you encountered and does not seem to introduce any new issues.